### ISSUE TITLE
ovs remove nodes failed: There are 2 items left in self.domains

### ISSUE THREAD

**Issue by JeffreyDevloo** [openvstorage/framework#776]
We encountered the following error during a 'ovs remove nodes' execution on our Hyperscale setup using:
- 3 controller nodes
- 1 computing node 
- 2 storage nodes

```
Update existing vPools
Restarting services
  [10.100.199.1] watcher-framework stopped
  [10.100.199.4] watcher-framework stopped
  [10.100.199.3] watcher-framework stopped
  [10.100.199.2] watcher-framework stopped
  [10.100.199.1] memcached restarted
  [10.100.199.4] memcached restarted
  [10.100.199.1] watcher-framework started
  [10.100.199.4] watcher-framework started
  [10.100.199.3] watcher-framework started
  [10.100.199.2] watcher-framework started
Avahi installed

+++ Announcing service +++

  [10.100.199.2] avahi-daemon restarted
Demote complete
    Removing node from model

+++ Running "remove" hooks +++



++++++++++++++++++++++++++++++++++++++++++++++++
+++  An unexpected error occurred:           +++
+++  There are 2 items left in self.domains  +++
++++++++++++++++++++++++++++++++++++++++++++++++
```

**Comment by kinvaris** [comment]
Detailed error:

```
2016-08-08 10:32:32 55700 +0200 - ovs-node1 - 6633/139886640301888 - lib/setup - 266 - ERROR - An unexpected error occurred:
Traceback (most recent call last):
  File "ovs/lib/setup.py", line 710, in remove_nodes
    storage_router.delete()
  File "ovs/dal/dataobject.py", line 700, in delete
    raise LinkedObjectException('There {0} left in self.{1}'.format(multi, key))
LinkedObjectException: There are 2 items left in self.domains
2016-08-08 10:32:32 55700 +0200 - ovs-node1 - 6633/139886640301888 - lib/setup - 267 - ERROR - There are 2 items left in self.domains
Traceback (most recent call last):
  File "ovs/lib/setup.py", line 710, in remove_nodes
    storage_router.delete()
  File "ovs/dal/dataobject.py", line 700, in delete
    raise LinkedObjectException('There {0} left in self.{1}'.format(multi, key))
LinkedObjectException: There are 2 items left in self.domains
```

**Comment by matthiasdeblock** [comment]
Also had an issue when removing a node: https://github.com/openvstorage/framework/issues/652

**Comment by khenderick** [comment]
Fixed by #801 in openvstorage-2.7.2-rev.3859.42b3488

**Comment by JeffreyDevloo** [comment]
I have not encountered the error during the removal on a similar test setup.
The issue on 'openvstorage-2.7.2-rev.3859.42b3488' seems to be fixed. Closing.
